docs/sql-ref-syntax-qry-select-transform.md

```diff
@@ -24,6 +24,15 @@ license: |
 The `TRANSFORM` clause is used to specify a Hive-style transform query specification 
```

Comment:
We don't have this file in DBR. I take it we don't support TRANSFORM?

Reply:
Just confirmed with @gatorsmile, we support `TRANSFORM` but do not encourage the customer to use it.